doc/articles/toc.yml

```diff
@@ -499,3 +499,9 @@
           href: uno-development/troubleshooting-source-generation.md
         - name: The XAML Trimming phase
           href: uno-development/Uno-UI-XAML-ResourceTrimming.md
+- name: Figma Plugin
```

Comment:
This will create a new top-level section in the TOC, which I don't think really makes sense. It's a how-to so it can go in the how-to section.

Reply:
I asked Luke to put it there as I thought of it as the plugin is separate from key Uno Platform. So, while the docs are relatable/cross-referenceable maybe, they deserve to stand alone and are more than a "How to" do something with Uno Platform itself.

Reply:
Fair enough. So something like this? I'd agree to that

>Overview
>Get Started
>Howt-tos and tutorials
>Development
>Reference
>Contributing to Uno
>Ad ons
>> Figma Plugin
>>>UnoFigma for Developers
>>>UnoFigma for Designers

>> Another Plugin
>> Another Plugin

Reply:
yep, that looks good to me.

Reply:
ok,lets do that instead @duke7553  please.

Reply:
@davidjohnoliver @sasakrsmanovic Done!